engine/execution/computation/computer/result_collector.go

```diff
@@ -259,5 +262,11 @@ func (collector *resultCollector) Finalize() (
 		return nil, err
 	}
 
+	collector.result.ExecutionResult = flow.NewExecutionResult(
+		collector.parentBlockExecutionResultID,
+		collector.result.ExecutableBlock.ID(),
+		collector.result.Chunks,
+		collector.result.ConvertedServiceEvents,
+		collector.result.ExecutionDataID)
```

Comment:
note that this id is currently incorrectly.  The id is correctly populated in #3927